CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOSUITests/CBLTestServer_iOSUITests.swift

```diff
@@ -0,0 +1,36 @@
+//
```

Comment:
same as above, remove it  as client tests cover API test coverage?

Reply:
These are autogenerated for the app. Its ok to leave them there. There are no tests in the file.